src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+
+    /**
+     * Information of the date of deadline.
+     */
+    private LocalDate date;
```

Comment:
Actually there isn't a really need to write javadocs for private fields but good that you have 👍